app/src/main/java/com/example/klisttarefa/Activity/view/activity/MainActivity.kt

```diff
@@ -28,35 +36,38 @@ class MainActivity : AppCompatActivity() {
         registrationTrue = ArrayList()
         registrationFalse = ArrayList()
 
+        registrationRepository = RegistrationRepository(RegistrationDataBase.getInstance(this).registrationDao)
 
-        val registrationOne = Registration(1,"Go to the gym", "Sport", true)
-        val registrationTwo = Registration(2,"Go to school", "study", false)
-
-
-        registration.add(registrationOne)
-        registration.add(registrationTwo)
 
-        for (itens in registration) {
-            if (itens.check == true) {
-                val layout = LinearLayoutManager(this)
-                binding.rvCompleted.layoutManager = layout
-                registrationTrue.add(itens)
-                adapter = AdapterList(registrationTrue, this)
-                binding.rvCompleted.adapter = adapter
-            } else {
-                val layout = LinearLayoutManager(this)
-                binding.rvIncomplete.layoutManager = layout
-                registrationFalse.add(itens)
-                adapter = AdapterList(registrationFalse, this)
-                binding.rvIncomplete.adapter = adapter
-            }
-
-        }
+      lifecycleScope.launch { listActivity() }
 
         binding.floatingActionButton.setOnClickListener {
             val intent = Intent(this, RegistrationActivity::class.java).apply {}
             startActivity(intent)
         }
 
     }
+
+    suspend fun listActivity(){
+        try {
+            registrationRepository.getAllProjet().forEach {
+                if (it.check == true) {
+                    val layout = LinearLayoutManager(this)
+                    binding.rvCompleted.layoutManager = layout
+                    registrationTrue.add(it)
+                    adapter = AdapterList(registrationTrue, this)
+                    binding.rvCompleted.adapter = adapter
+                } else {
+                    val layout = LinearLayoutManager(this)
```

Comment:
Não precisa criar o `val layout = LinearLayoutManager(this)` duas vezes, deixa fora do bloco